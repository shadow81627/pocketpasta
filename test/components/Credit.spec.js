import Component from '@/components/Layout/Credit/landing-hero-image-credit';
import { shallowMount, createLocalVue } from '@vue/test-utils';
import Vuex from 'vuex';

const localVue = createLocalVue();

localVue.use(Vuex);

describe('Credit', () => {
  const factory = () => shallowMount(Component, {});

  test('mounts properly', () => {
    const wrapper = factory();
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  test('renders properly', () => {
    const wrapper = factory();
    expect(wrapper.html()).toMatchSnapshot();
  });
});
