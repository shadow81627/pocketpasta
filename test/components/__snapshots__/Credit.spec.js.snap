// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Credit renders properly 1`] = `"<a href=\\"https://unsplash.com/@icons8?utm_medium=referral&amp;utm_campaign=photographer-credit&amp;utm_content=creditBadge\\" target=\\"_blank\\" rel=\\"noopener noreferrer\\" title=\\"Download free do whatever you want high-resolution photos from Icons8 team\\" style=\\"background-color: black; color: white; text-decoration: none; padding: 4px 6px; font-family: -apple-system, BlinkMacSystemFont, 'San Francisco',; font-size: 12px; font-weight: bold; line-height: 1.2; display: inline-block; border-radius: 3px;\\"><span class=\\"unsplash-logo\\" style=\\"display: inline-block; padding: 2px 3px;\\"><svg xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 32 32\\"><title>unsplash-logo</title> <path d=\\"M10 9V0h12v9H10zm12 5h10v18H0V14h10v9h12v-9z\\"></path></svg></span><span style=\\"display: inline-block; padding: 2px 3px;\\">Icons8 team</span></a>"`;
